src/MOIWrapper.jl

```diff
@@ -86,54 +86,40 @@ LQOI.backend_type(::Optimizer, ::MOI.Nonnegatives)         = Cchar('>')
 function LQOI.change_variable_bounds!(model::Optimizer,
           columns::Vector{Int}, new_bounds::Vector{Float64},
           senses::Vector{Cchar})
-    number_lower_bounds = count(x->x==Cchar('L'), senses)
-    lower_cols   = fill(0, number_lower_bounds)
-    lower_values = fill(0.0, number_lower_bounds)
-    number_upper_bounds = count(x->x==Cchar('U'), senses)
-    upper_cols   = fill(0, number_upper_bounds)
-    upper_values = fill(0.0, number_upper_bounds)
-    lower_index = 1
-    upper_index = 1
     for (column, bound, sense) in zip(columns, new_bounds, senses)
         if sense == Cchar('L')
-            lower_cols[lower_index]   = column
-            lower_values[lower_index] = bound
-            lower_index += 1
+            set_dblattrelement!(model.inner, "LB", column, bound)
```

Comment:
what about: http://www.gurobi.com/documentation/8.0/refman/c_grbsetdblattrlist.html ?

Reply:
Got it,
Did you benchmark with 1000 variables?
If the new method is better for 1000 variables I definatly vote for it. (And you can merge). Otherwise I would like more info.

Reply:
I've added benchmarks to the top post. The new method is actually better for all sizes < 10^4... I must have conducted the previous benchmarks on different LQOI commits or something.